R/db-sql.R

```diff
@@ -257,11 +275,11 @@ sql_select.DBIConnection <- function(con, select, from, where = NULL,
 
 #' @rdname db-sql
 #' @export
-sql_query_join <- function(con, x, y, vars, type = "inner", by = NULL, na_matches = FALSE, ...) {
+sql_query_join <- function(con, x, y, vars, type = "inner", by = NULL, na_matches = FALSE, ..., lvl = 0) {
```

Comment:
If a package has provided their own `sql_query_join()` method, this will still work, just the indenting will be incorrect, right?

Reply:
Yes, there should be no problem. The `lvl` argument is just for indenting and the output is the same as before apart from indent.